geode-core/src/main/java/org/apache/geode/cache/client/internal/pooling/ConnectionManagerImpl.java

```diff
@@ -308,24 +308,49 @@ public Connection borrowConnection(long acquireTimeout)
    * destroyed when returned to the pool.
    */
   @Override
-  public PooledConnection borrowConnection(ServerLocation server,
-      boolean onlyUseExistingCnx) throws AllConnectionsInUseException, NoAvailableServersException {
+  public PooledConnection borrowConnection(ServerLocation server, long acquireTimeout,
+      boolean onlyUseExistingCnx)
+      throws AllConnectionsInUseException, NoAvailableServersException, ServerOperationException {
```

Comment:
I don't see where this is throwing ServerOperationException.  It throws its superclass, ServerConnectivityException.

Reply:
Updated